localstorage.js

```diff
@@ -0,0 +1,30 @@
+// formData object
+let formData = {
+  fullname: '',
+  email: '',
+  description: '',
+};
+/* eslint-disable */
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to solve linter errors instead of disabling the inters.